### PR TITLE
assig2

### PR DESCRIPTION
**Pull request by xiyaa**

### CODE REVIEW

**Comment by ocorcho** [comment]
Todo bien, pero no puedes eliminar el README.md

File: Assignment4/notebooks/task06.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1btcAP3jsLeBGJxbh8N82O94i6uAroI8m
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/notebooks/task06.py:12]: This won't run in Python. You need to comment it out

File: Assignment4/notebooks/task07.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  initNs={"rdfs": RDFS, "ns": ns}
+)
+
+# Visualize the results
+for r in g.query(q2):
+  print(r.Subject)
+for r in g.query(q3):
+  q4 = prepareQuery('''
+  SELECT ?Subject WHERE { 
+    ?Subject rdf:type <r>
```
Comment (dgarijo) [inline Assignment4/notebooks/task07.py:76]: Malformed query. You are asking for the URI <r>, which should not exist.

File: Assignment4/notebooks/task07.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    ?Subject ?prop ?val
+  }
+  ''',
+  initNs={"rdf": RDF, "ns": ns}
+)
+
+q5 = prepareQuery('''
+  SELECT ?Subject ?prop ?val WHERE { 
+    ?Subject rdf:type <http://somewhere#Researcher> .
+    ?Subject ?prop ?val
```
Comment (dgarijo) [inline Assignment4/notebooks/task07.py:113]: Not correct, you need to extract the subclasses in the query itself

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please address the problems in task 7

File: Assignment4/notebooks/task07.py (hunk 3 above)
Comment (xiyaa) [reply Assignment4/notebooks/task07.py]: ok

**Comment by ocorcho** [comment]
Assignment 2 is ok. Now @dgarijo can check assignment on Python.

File: Assignment4/xiyaa-c20c030/task07.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+"""
+
+# TO DO WITH RDFLIB
+for s,p,o in g.triples((None, RDF.type, ns.Person)):
+  print(s)
+for s,p,o in g.triples((None, RDFS.subClassOf, ns.Person)):
+  subclass = s
+for s,p,o in g.triples((None, RDF.type, subclass)):
+  print(s)
```
Comment (dgarijo) [inline Assignment4/xiyaa-c20c030/task07.py:53]: This is not asked, as you are printing all individuals

**Pull request merged**